src/enclave/enclave_event_channel.c

```diff
@@ -145,18 +150,36 @@ void initialize_enclave_event_channel(
 
     evt_chn_lock = (struct ticketlock**)oe_calloc(
         evt_channel_num, sizeof(struct ticketlock*));
+    if (!evt_chn_lock)
+    {
+        sgxlkl_fail("Could not allocate memory for evt_chn_lock\n");
+    }
 
     vio_tasks =
         (struct lthread**)oe_calloc(evt_channel_num, sizeof(struct lthread*));
+    if (!vio_tasks)
```

Comment:
This pattern is repeated a lot.  Is it worth adding a helper along the lines of:
```c
void *oe_alloc_or_die(size_t size, size_t count, const char *fail_msg)
{
  void *ptr = oe_calloc(size, count);
  if (ptr == NULL)
  {
    sgxlkl_fail(fail_msg);
  }
  return ptr;
}
```

Reply:
Good idea. Where shall we put this?

Reply:
+1, I just said the same in my review. I was thinking along the lines of `#define CHECKED_CALLOC(X,Y) if ((X = oe_calloc(1, (Y)) == NULL) oe_abort();`

Reply:
Done.